backend/src/main/java/com/woowacourse/friendogly/pet/domain/SizeType.java

```diff
@@ -15,4 +17,10 @@ public static SizeType toSizeType(String sizeType) {
             throw new FriendoglyException("존재하지 않는 SizeType 입니다.");
         }
     }
+
+    public static Set<SizeType> toSizeTypes(Set<String> sizeType) {
+        return sizeType.stream()
+                .map(SizeType::toSizeType)
+                .collect(Collectors.toSet());
```

Comment:
```suggestion
    public static Set<SizeType> toSizeTypes(Set<String> sizeType) {
        return sizeType.stream()
                .map(SizeType::toSizeType)
                .collect(Collectors.toUnmodifiableSet());
```
불변 set으로 리턴하는 건 어떨까요?!

Reply:
놓친 부분이네요 approve 주셔서 추후에 수정하겠습니다.